docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Maybe you could also explain above, why setting <code>seed=0</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
This is described in the tutorials that are linked just above so I am going to leave it to keep the notebook as lean as possible and only related to JAX+DALI.